src/Chapter06/Listing06.35A.RequiredProperties.cs

```diff
@@ -0,0 +1,67 @@
+#if NET7_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_35A
+{
+    #region INCLUDE
+    public class Book
+    {
+        string? _Title;
+        #region HIGHLIGHT
+        public required string Title
+        #endregion HIGHLIGHT
+        {
+            get
+            {
+                return _Title!;
+            }
+            set
+            {
+                ArgumentException.ThrowIfNullOrEmpty(
+                    value = value?.Trim()!);
```

Comment:
Is it pertinent to the example that we also check for whitespace? It ends up making the code a lot more complicated than it needs to be.